### PR TITLE
Add in Ruby

### PR DESCRIPTION
**Pull request by andymckay**
This adds in support for Ruby as an out of the box template. I just copied this over from the to-do directory in the existing templates. 

Is this cool to use @bryanmacfarlane?

cc @mscoutermarsh

### CODE REVIEW

**Review by mscoutermarsh — APPROVED** [review]
🏅

**Review by damccorm — APPROVED** [review]
LGTM

**Review by themar7777 — APPROVED** [review]

**Comment by KimJi55** [comment]
Still confused, need to take a moment to understand Overall, it's great 👍🏼👍🏼

**Review by AiricAlchemy — APPROVED** [review]

File: ci/ruby.properties.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Ruby",
+    "description": "Build and test a Ruby project with Rake.",
+    "iconName": "ruby",
```
[the remark this answers is not in the document]
Comment (plunker2025) [reply ci/ruby.properties.json:4]: > ```suggestion
>     "iconName": "ruby.svg",
> ```

**Review by plunker2025 — APPROVED** [review]

**Comment by Mzachky** [comment]
https://github.com/Mzachky/starter-workflows/actions/workflows/sync-ghes.yaml

**Review by Mzachky — APPROVED** [review]

**Pull request merged**